### PR TITLE
Use shields.io compliant badge for SauceLabs

### PR DESCRIPTION
**Pull request by FezVrasta**
Just a small change to make the readme nicer.

image

(I'm pushing to master since this change affects only the readme file)

### CODE REVIEW

**Comment by FezVrasta** [comment]
The badge I used supports multiple browsers and versions.

image

**Comment by yyx990803** [comment]
Ah cool. It's showing Android only because I recently ran a single-browser test.

File: README.md [hunk 1]
```diff
@@ -6,8 +6,7 @@
   <a href="https://www.npmjs.com/package/vue"><img src="https://img.shields.io/npm/dt/vue.svg" alt="Downloads"></a>
   <a href="https://www.npmjs.com/package/vue"><img src="https://img.shields.io/npm/v/vue.svg" alt="Version"></a>
   <a href="https://www.npmjs.com/package/vue"><img src="https://img.shields.io/npm/l/vue.svg" alt="License"></a>
-  <br>
-  <a href="https://saucelabs.com/u/vuejs"><img src="https://saucelabs.com/browser-matrix/vuejs.svg" alt="Sauce Test Status"></a>
+  <a href="https://saucelabs.com/u/vuejs"><img src="https://badges.herokuapp.com/sauce/vuejs" alt="Sauce Test Status"></a>
```
Comment (yyx990803) [inline README.md]: Let's preserve the `<br>` here so that browser matrix is on a separate line.

**Comment by FezVrasta** [comment]
done

**Comment by posva** [comment]
This is very subjective, but wouldn't this be too many badges?
I really think the matrix version is easier to read. The line break wouldn't suffice imo to make the badge equivalent readable enough

**Comment by FezVrasta** [comment]
I personally don't like the matrix because it's very big and takes a lot of vertical space. The badges are nice and people are used to them.

**Comment by posva** [comment]
I see. So many badges on a single line are harder to read, even if we're used to reading badges. I personally prioritise readability over vertical space on a readme

**Comment by FezVrasta** [comment]
Still personal preference, but to me it's easier to read: "Browser: version, version, version" instead of  the matrix style. Especially because in the matrix style you have also the OS version that confuses a lot, for example myself get confused often by the windows version and the IE version 🙄

**Comment by yyx990803** [comment]
On a second thought, I still find the matrix to provide a better at-a-glance view of browser compatibility than badges. Thanks for the PR though!

**Pull request closed without merging**